roles/libvirt_manager/README.md

```diff
@@ -64,6 +64,10 @@ cifmw_libvirt_manager_configuration:
       target: (Hypervisor hostname you want to deploy the family on. Optional)
       uefi: (boolean, toggle UEFI boot. Optional, defaults to false)
       bootmenu_enable: (string, toggle bootmenu. Optional, defaults to "no")
+      ip_address: (dict, having static IP address information. Optional)
```

Comment:
Instead of this, can we integrate with the `cloud_init` role instead?
 It can will allow us to skip the whole "virt-sysprep" step and provide a much richer network (and other) config using a single format independent of OS relese (fedrora, centos/rhel 10 etc).
It also supports creating the user, and inject SSH keys, give network interfaces nice names like `ocpbm0`, `osp_trunk0` etc.

Example below and here: https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/nat64_appliance/tasks/deploy.yml#L61-L99

```
- name: Create the config-drive ISO for the IPI provisioner node
  vars:
    cifmw_config_drive_iso_image: "{{ _workload_dir }}/{{ _host }}.iso"
    cifmw_config_drive_uuid: "{{ _hist_uuid }}"
    cifmw_config_drive_name: "{{ _host }}"
    cifmw_config_drive_hostname: "{{ _hostname }}"
    cifmw_config_drive_userdata:
      users:
        - name: zuul
          ssh_authorized_keys:
            - "{{ _ssh_key.public_key }}"
          sudo: "ALL=(ALL) NOPASSWD:ALL"
          lock_passwd: false
          plain_text_passwd: fooBar
    cifmw_config_drive_networkconfig:
      network:
        version: 2
        ethernets:
          management:
            match:
              macaddress: "{{ _mac_address }}"
            set-name: mgmt0
            dhcp4: true
  ansible.builtin.include_role:
    name: config_drive
```